setup.py

```diff
@@ -30,12 +34,16 @@ def get_version():
     'pytest-asyncio>=0.5.0',
     'pytest-cov>=2.3.1',
     'logbook>=1.0.0',
+    'flake8>=3.0.4',
+    'isort>=4.2.5',
+    'collective.checkdocs>=0.2',
+    'Pygments>=2.1.3'  # required by checkdocs
```

Comment:
You are right about that. But I'd rather support up-to-date versions as it mitigates conflicts with other packages. And if something breaks, either Travis or someone else will report us an issue. Having said that, can we configure Travis to run every week or so regardless of commits?